java/arcs/core/data/testdata/BUILD

```diff
@@ -16,3 +17,8 @@ arcs_manifest_proto(
     src = "Example.arcs",
     visibility = ["//visibility:public"],
 )
+
+proto2schema(
+    name = "example_generation",
+    srcs = [":example"],
```

Comment:
You could consider renaming `example` to something like `example_pb` so that it's easier to tell what it generates.